app/Filament/Organizations/Resources/BeneficiaryResource/Pages/ViewBeneficiary.php

```diff
@@ -110,6 +109,10 @@ protected function identitySectionSection(): Section
                     ->icon('heroicon-o-phone')
                     ->url(fn ($state) => "tel:{$state}"),
 
+                TextEntry::make('email')
+                    ->label(__('beneficiary.section.identity.labels.email'))
+                    ->icon('heroicon-o-envelope'),
```

Comment:
Email validation

Reply:
it's view page, in edit page i added email type